src/app/models/node.ts

```diff
@@ -11,20 +11,29 @@ export class Node {
     id: string;
     identifiers: Array<string>;
     name: string;
-    obms: string[];
+    obms: Obms[];
     tags: string;
     pollers: string;
     relations: string[];
     sku?: string;
     type: string;
     workflows: string;
-    ibms: string[];
+    ibms: Ibms[];
 
     discoveredTime: string;
     manufacturer?: string;
     model: string;
 }
 
+class Obms {
```

Comment:
this can be an interface no need to be a class

Reply:
completely agree!